README.md

```diff
@@ -6,168 +6,208 @@ level: 100
 service: ASP.NET Core Web App
 endpoint: AAD V2
 ---
-# Integrating Azure AD V2 into an ASP.NET Core web app
+# Integrating Azure AD V2 into an ASP.NET Core web app that calls the Microsoft Graph
 
 ![Build badge](https://identitydivision.visualstudio.com/_apis/public/build/definitions/a7934fdd-dcde-4492-a406-7fad6ac00e17/514/badge)
 
 ## Scenario
 
-This sample shows how to build a .NET Core MVC Web app that uses OpenID Connect to sign in personal accounts (including outlook.com, live.com, and others) as well as work and school accounts from any company or organization that has integrated with Azure Active Directory. It leverages the ASP.NET Core OpenID Connect middleware.
+This sample shows how to build a .NET Core MVC Web app that uses OpenID Connect to sign in personal accounts (including outlook.com, live.com, and others) as well as work and school accounts from any company or organization that has integrated with Azure Active Directory The Web App then  calls the Microsoft Graph on behalf of the signed-in user. The sample leverages the ASP.NET Core OpenID Connect middleware and [MSAL.NET](http://aka.ms/msalnet)
 
-![Sign-in with Azure AD](ReadmeFiles/sign-in.png)
-
-An on demand video was created for the Build 2018 event, featuring this scenario and this sample. See the video [Building Web App Solutions With Authentication](https://channel9.msdn.com/Events/Build/2018/THR5001), and the associated [PowerPoint deck](http://video.ch9.ms/sessions/c1f9c808-82bc-480a-a930-b340097f6cc1/SigninworkandschoolMSAusersWebApp.pptx)
+> This is the second tutorial, building on top of the previous one (in the master branch) showing how to add sign-in to a Web application. This ones adds the part which is about calling the Microsoft Graph in a controller on behalf of the user
 
 ## How to run this sample
 
 To run this sample:
 
-> Pre-requisites: Install .NET Core (for example for Windows) by following the instructions at [.NET and C# - Get Started in 10 Minutes](https://www.microsoft.com/net/core). In addition to developing on Windows, you can develop on [Linux](https://www.microsoft.com/net/core#linuxredhat), [Mac](https://www.microsoft.com/net/core#macos), or [Docker](https://www.microsoft.com/net/core#dockercmd).
+> Pre-requisites: go through how to run this sample from the master branch. This page only shows the increment.
```

Comment:
[this sample from the master branch](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2). This page shows the incremental change to demonstrate how to call the Microsoft Graph API on behalf of a user that have successfully signed in to the web app.

Reply:
Good suggestion.